### PR TITLE
[AIRFLOW-3519] Fix example http operator

### PR DESCRIPTION
**Pull request by feluelle**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-3519
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

This PR fixes the sensor in the example http operator that searches for a string in a byte-like object called response.content.
Now it searches in the decoded response object called response.text.

**NOTE:** This PR also fixes issue https://issues.apache.org/jira/browse/AIRFLOW-450. This ticket was already marked as resolved but it actually wasn't resolved.

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

File: airflow/example_dags/example_http_operator.py [hunk 1]
```diff
@@ -92,7 +92,7 @@
     http_conn_id='http_default',
     endpoint='',
     request_params={},
-    response_check=lambda response: True if "Google" in response.content else False,
```
Comment (kaxil) [inline airflow/example_dags/example_http_operator.py:95]: response.content works as well. Does it error for you @feluelle ?

**Review by kaxil — CHANGES_REQUESTED** [review]

File: airflow/example_dags/example_http_operator.py (hunk 1 above)
Comment (feluelle) [reply airflow/example_dags/example_http_operator.py:95]: Yes. `response.content` is a byte string and not a decoded string like `response.text`.
So either `b"Google" in response.content` would work or `"Google" in response.text` . See these Jira tickets: https://issues.apache.org/jira/browse/AIRFLOW-3519 and https://issues.apache.org/jira/browse/AIRFLOW-450

Comment (kaxil) [reply airflow/example_dags/example_http_operator.py:95]: Makes sense. Thanks @feluelle

**Review by kaxil — APPROVED** [review]

File: airflow/example_dags/example_http_operator.py (hunk 1 above)
Comment (feluelle) [reply airflow/example_dags/example_http_operator.py:95]: ..and I thought `"Google" in response.text` makes more sense (to me) and you do not need to do the byte conversion by yourself. :)

**Pull request merged**